submissions/models.py

```diff
@@ -205,6 +205,25 @@ def create_reset_score(cls, student_item):
             reset=True,
         )
 
+    @classmethod
+    def create_override_score(cls, student_item_model, points_possible, override_grade):
```

Comment:
Is there a reason we're passing the `points_possible` before passing in
`points_earned`? This may be personal bias, but I'm used to seeing
earned points before possible points; e.g., "5 out of 10" and "5/10".

Also, it looks like the API's existing `set_score` method takes
`points_earned` first, followed by `points_possible`.

For consistency, could we update the signature to:
```python
def create_override_score(cls, student_item, points_earned, points_possible):
    pass
```